### PR TITLE
Add cpu flags infos to help to debug

### PR DESCRIPTION
**Pull request by ggardet**
possible problems related to supported/unsupported instructions.

### CODE REVIEW

File: lib/OpenQA/WebAPI/Controller/API/V1/Worker.pm [hunk 1]
```diff
@@ -177,15 +177,16 @@ sub create {
...
     return $self->reply->validation_error({format => 'json'}) if $validation->has_error;
 
     my $host     = $validation->param('host');
     my $instance = $validation->param('instance');
     my $job_ids  = $validation->every_param('job_id');
     my $caps     = {};
     $caps->{$_} = $validation->param($_)
-      for qw(cpu_arch cpu_modelname cpu_opmode mem_max isotovideo_interface_version websocket_api_version worker_class);
+      for
+      qw(cpu_arch cpu_modelname cpu_opmode cpu_flags mem_max isotovideo_interface_version websocket_api_version worker_class);
```
Comment (okurz) [inline lib/OpenQA/WebAPI/Controller/API/V1/Worker.pm:189]: I think we can do better for readability here :)

Comment (ggardet) [reply lib/OpenQA/WebAPI/Controller/API/V1/Worker.pm:189]: Yes, but this how tidy want I write it.

**Comment by ggardet** [comment]
Updated. Not sure why CI fails, though.

File: lib/OpenQA/WebAPI/Controller/API/V1/Worker.pm (hunk 1 above)
Comment (okurz) [reply lib/OpenQA/WebAPI/Controller/API/V1/Worker.pm:189]: yes, but you could for example define a variable to use here or a list of line-wrapped constants at the top of the file.

**Comment by okurz** [comment]
https://app.circleci.com/pipelines/github/os-autoinst/openQA/2675/workflows/bdb53a72-2b5c-403e-99fe-f3eb8e0b7d45/jobs/25137/steps shows

```
#   Failed test 'capabilities contain expected information'
    #   at t/24-worker-overall.t line 121.
    #     Structures begin differing at:
    #          $got->[1] = 'cpu_flags'
    #     $expected->[1] = 'host'
    # {
    #   'cpu_arch' => 'x86_64',
    #   'cpu_flags' => 'fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss ht syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon rep_good nopl xtopology nonstop_tsc cpuid aperfmperf tsc_known_freq pni pclmulqdq monitor ssse3 fma cx16 pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single pti fsgsbase tsc_adjust bmi1 hle avx2 smep bmi2 erms invpcid rtm mpx avx512f avx512dq rdseed adx smap clflushopt clwb avx512cd avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves ida arat pku ospke',
    #   'host' => 'a39aefa2f72e',
    #   'instance' => 1,
    #   'isotovideo_interface_version' => 0,
    #   'mem_max' => 70340,
    #   'websocket_api_version' => 1,
    #   'worker_class' => 'qemu_i386,qemu_x86_64'
    # }
    # Looks like you failed 1 test of 3.
 ```

This is exactly related to your changes. I recommend you reproduce the problem by running the test locally and then adjust https://github.com/os-autoinst/openQA/blob/master/t/24-worker-overall.t#L125 accordingly.

**Comment by ggardet** [comment]
> This is exactly related to your changes. I recommend you reproduce the problem by running the test locally and then adjust https://github.com/os-autoinst/openQA/blob/master/t/24-worker-overall.t#L125 accordingly.

When test is fixed for `x86_64` by adding the new `cpu_flags`, it starts to fail for `aarch64`.
Apparently in CI, `x86_64` `lscpu` outputs `Flags`, but not `aarch64`? Or am I missing something?

**Comment by Martchus** [comment]
I don't know how `lscpu` generally behaves under `aarch64`. If a field is not generally present you can just exclude it from the test completely (simply delete the key from `$capabilities` before the `is_deeply` check). I suppose that's better than having this (unimportant) test failing under some platforms.

**Comment by ggardet** [comment]
> I don't know how `lscpu` generally behaves under `aarch64`. If a field is not generally present you can just exclude it from the test completely (simply delete the key from `$capabilities` before the `is_deeply` check). I suppose that's better than having this (unimportant) test failing under some platforms.

It should be there. Which Leap version is used in CI?

**Pull request merged**